README.md

```diff
@@ -62,6 +66,41 @@ Launch the application with the following commands:
     ros2 launch hello_world_simulation empty_world.launch.py
     ```
 
+Note that when running robot applications on a robot, `use_sim_time` should be set to `false` (which is the default value in `deploy_rotate.launch.py`). When running robot applications along with simulation applications, `use_sim_time` should be set to `true` for both applications (which is the default value in both `rotate.launch.py` and `empty_word.launch.py`).
+
+When running simulation applications, run command with `gui:=true` to run gazebo client for visualization
+
```

Comment:
Noting unrelated minor changes to the README. Next time, let's keep the commits separate for clearer history.

Reply:
Sorry for the confusion. I will make sure they are in separate PRs next time.